docs/tutorials/p2p.md

```diff
@@ -106,61 +96,52 @@ It pretty simply acknowledges the session request, accepts it, then sends a hand
 ## Reading P2P Packets
 ==}
 
-Inside that handshake there was a call to the `read_p2p_packet()` function which does this:
+Inside that handshake there was a call to the `read_messages()` function which does this:
```

Comment:
The lobby tutorials would need to be updated to reflect this.  Preferably with tabbed versions of code for each class.  Also the lobby tutorial needs update as the handshake process has changed a bit in the newest, suggested version from Skillet.